example/61_contraction_multi_ABD/contraction_multi_ABD_xdl_fp16.cpp

```diff
@@ -0,0 +1,328 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
AFAIK, we are supposed to write only the file creation date when we add new files.
```suggestion
// Copyright (c) 2023, Advanced Micro Devices, Inc. All rights reserved.
```